SCClassLibrary/Common/Audio/FFT2.sc

```diff
@@ -56,19 +56,23 @@ Convolution3 : UGen {
 
 
 //jensen andersen inspired FFT feature detector
-PV_JensenAndersen : PV_ChainUGen {
+JensenAndersen : PV_ChainUGen {
```

Comment:
Doesn't this end up with the wrong UGen name for the server? The UGen is still called `PV_JensenAndersen`, or not? Doesn't the class file then have to provide an explicit name somewhere?